deploy/scripts/src/mobile/utils/functions-mobile-journey.ts

```diff
@@ -1,31 +1,12 @@
 import { check, group } from 'k6'
-import http, { type CookieJar, type Response } from 'k6/http'
+import http, { type Response } from 'k6/http'
```

Comment:
Move everything in utils except this file into /k6, and move readme into /mobile